src/lib.rs

```diff
@@ -61,3 +63,33 @@ unsafe impl GlobalAlloc for LibcAlloc {
         libc::realloc(ptr as *mut c_void, new_size) as *mut u8
     }
 }
+
+#[cfg(target_family = "windows")]
+mod win_crt;
+
+#[cfg(target_family = "windows")]
+unsafe impl GlobalAlloc for LibcAlloc {
+    #[inline]
+    unsafe fn alloc(&self, layout: Layout) -> *mut u8 {
+        return win_crt::_aligned_malloc(layout.size(), layout.align()) as *mut u8;
+    }
+
+    #[inline]
+    unsafe fn dealloc(&self, ptr: *mut u8, _layout: Layout) {
+        return win_crt::_aligned_free(ptr as *mut c_void);
+    }
+
+    #[inline]
+    unsafe fn alloc_zeroed(&self, layout: Layout) -> *mut u8 {
+        let ptr = self.alloc(layout);
+        if !ptr.is_null() {
+            ptr::write_bytes(ptr, 0, layout.size());
```

Comment:
Please duplicate the comment from `alloc_zeroed` above regarding why this manual zeroing is required.

Reply:
(bump)